portafly/doc/ux_specs/audience/accounts/Listing.md

```diff
@@ -0,0 +1,21 @@
+# Audience > Accounts > Listing
+
+* [Mockups](https://marvelapp.com/55343de/screen/67190788)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4725)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4724)
```

Comment:
Will all the Jiras be open to the public?

Reply:
it looks like they currently are open to the public. I can access them without being logged in.
if you think that linking to the JIRA issues is not necessary here, I'm good with that.